src/Cms/Item.php

```diff
@@ -26,6 +26,11 @@ class Item
 
 	public const ITEMS_CLASS = Items::class;
 
+	/**
+	 * @var \Kirby\Cms\Field|null
+	 */
+	protected $field;
```

Comment:
Could we start typing new properties? I know we're running into problems with our existing ones - as our docs and code base aren't as consistent as they would need to be. But maybe we could start with new ones?